website/content/en/docs/about/concepts.md

```diff
@@ -54,6 +54,43 @@ A **pipeline** is a [directed acyclic graph][dag] of [components]. Each componen
 
 {{< jump "/docs/about/under-the-hood/architecture/pipeline-model" >}}
 
+
+## Buffers
+
+Sinks will try to send events as fast as possible. If they are unable to keep up, they have a configurable buffer that will hold events until they can be sent.
+By default, Vector uses an in-memory buffer, but a disk-buffer is also available. Once a buffer fills up, the behavior is configurable.
+
+`buffer.when_full = block`
+This is the default behavior. When a buffer fills up, backpressure will be applied to previous components in the graph.
+
+`buffer.when_full = drop_newest`
+When a buffer fills up, new events will be dropped. This does NOT provide backpressure.
+
+## Backpressure
```

Comment:
I'm very happy to have something to point people too for a reference.

Could we add a link to this section to the cue reference for `when_full: "block"` (e.g. https://vector.dev/docs/reference/configuration/sinks/vector/#buffer.when_full).